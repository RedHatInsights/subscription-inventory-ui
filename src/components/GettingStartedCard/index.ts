import GettingStartedCard from './GettingStartedCard';

export default GettingStartedCard;
