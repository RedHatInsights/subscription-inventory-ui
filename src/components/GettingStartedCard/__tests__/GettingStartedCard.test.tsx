import React from 'react';
import { fireEvent, render } from '@testing-library/react';
import GettingStartedCard from '../GettingStartedCard';

describe('GettingStartedCard', () => {
  it('renders a button', () => {
    render(<GettingStartedCard />);

    expect(document.body).toMatchSnapshot();
  });

  it('renders the card when the toggle is clicked', async () => {
    const { getByTestId } = render(<GettingStartedCard />);

    fireEvent.click(getByTestId('resources-toggle-button'));
    expect(document.body).toMatchSnapshot();
  });
});
