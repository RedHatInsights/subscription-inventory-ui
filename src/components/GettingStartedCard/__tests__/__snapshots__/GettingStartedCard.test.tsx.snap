// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GettingStartedCard renders a button 1`] = `
<body>
  <div>
    <article
      class="pf-c-card"
      data-ouia-component-id="OUIA-Generated-Card-1"
      data-ouia-component-type="PF4/Card"
      data-ouia-safe="true"
      id="getting-started-card"
    >
      <div
        class="pf-c-card__header"
      >
        <div
          class="pf-c-card__header-toggle"
        >
          <button
            aria-disabled="false"
            aria-expanded="false"
            aria-label="Actions"
            aria-labelledby="titleId resources-toggle-button"
            class="pf-c-button pf-m-plain"
            data-ouia-component-id="OUIA-Generated-Button-plain-1"
            data-ouia-component-type="PF4/Button"
            data-ouia-safe="true"
            data-testid="resources-toggle-button"
            id="resources-toggle-button"
            type="button"
          >
            <span
              class="pf-c-card__header-toggle-icon"
            >
              <svg
                aria-hidden="true"
                fill="currentColor"
                height="1em"
                role="img"
                style="vertical-align: -0.125em;"
                viewBox="0 0 256 512"
                width="1em"
              >
                <path
                  d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"
                />
              </svg>
            </span>
          </button>
        </div>
        <div
          class="pf-l-level pf-m-gutter"
        >
          <div
            class="pf-c-card__title"
            id="getting-started-title"
          >
            Getting started resources
          </div>
        </div>
      </div>
    </article>
  </div>
</body>
`;

exports[`GettingStartedCard renders the card when the toggle is clicked 1`] = `
<body>
  <div>
    <article
      class="pf-c-card pf-m-expanded"
      data-ouia-component-id="OUIA-Generated-Card-2"
      data-ouia-component-type="PF4/Card"
      data-ouia-safe="true"
      id="getting-started-card"
    >
      <div
        class="pf-c-card__header"
      >
        <div
          class="pf-c-card__header-toggle"
        >
          <button
            aria-disabled="false"
            aria-expanded="true"
            aria-label="Actions"
            aria-labelledby="titleId resources-toggle-button"
            class="pf-c-button pf-m-plain"
            data-ouia-component-id="OUIA-Generated-Button-plain-2"
            data-ouia-component-type="PF4/Button"
            data-ouia-safe="true"
            data-testid="resources-toggle-button"
            id="resources-toggle-button"
            type="button"
          >
            <span
              class="pf-c-card__header-toggle-icon"
            >
              <svg
                aria-hidden="true"
                fill="currentColor"
                height="1em"
                role="img"
                style="vertical-align: -0.125em;"
                viewBox="0 0 256 512"
                width="1em"
              >
                <path
                  d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"
                />
              </svg>
            </span>
          </button>
        </div>
        <div
          class="pf-c-card__title"
          id="getting-started-title"
        >
          Getting started resources
        </div>
      </div>
      <div
        class="pf-c-card__expandable-content"
      >
        <div
          class="pf-c-card__body"
        >
          <div
            class="pf-l-grid pf-m-all-6-col-on-md pf-m-all-3-col-on-lg pf-m-gutter"
          >
            <div
              class="pf-l-flex pf-m-space-items-lg pf-m-column pf-m-align-items-flex-start"
            >
              <div
                class="pf-l-flex pf-m-space-items-sm pf-m-grow pf-m-column pf-m-align-items-flex-start"
              >
                <span
                  class="pf-c-label pf-m-blue"
                >
                  <span
                    class="pf-c-label__content"
                  >
                    <span
                      class="pf-c-label__icon"
                    >
                      <svg
                        aria-hidden="true"
                        fill="currentColor"
                        height="1em"
                        role="img"
                        style="vertical-align: -0.125em;"
                        viewBox="0 0 384 512"
                        width="1em"
                      >
                        <path
                          d="M336 64h-80c0-35.3-28.7-64-64-64s-64 28.7-64 64H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h288c26.5 0 48-21.5 48-48V112c0-26.5-21.5-48-48-48zM192 40c13.3 0 24 10.7 24 24s-10.7 24-24 24-24-10.7-24-24 10.7-24 24-24zm121.2 231.8l-143 141.8c-4.7 4.7-12.3 4.6-17-.1l-82.6-83.3c-4.7-4.7-4.6-12.3.1-17L99.1 285c4.7-4.7 12.3-4.6 17 .1l46 46.4 106-105.2c4.7-4.7 12.3-4.6 17 .1l28.2 28.4c4.7 4.8 4.6 12.3-.1 17z"
                        />
                      </svg>
                    </span>
                    Activate a subscription
                  </span>
                </span>
                <p>
                  Activate a subscription purchased from a third party.
                </p>
                <ul
                  class="pf-c-list pf-m-plain"
                >
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-1"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://access.redhat.com/subscriptions/activate/redhat"
                      target="_blank"
                    >
                      Subscription Activation
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-2"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://access.redhat.com/subscriptions/activate/dell"
                      target="_blank"
                    >
                      Activate a Dell Service Tag
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                </ul>
              </div>
              <a
                aria-disabled="false"
                class="pf-c-button pf-m-link pf-m-inline"
                data-ouia-component-id="OUIA-Generated-Button-link-3"
                data-ouia-component-type="PF4/Button"
                data-ouia-safe="true"
                href="https://access.redhat.com/support/contact/customerService"
              >
                Contact Red Hat Customer Service
                <span
                  class="pf-c-button__icon pf-m-end"
                >
                  <svg
                    aria-hidden="true"
                    fill="currentColor"
                    height="1em"
                    role="img"
                    style="vertical-align: -0.125em;"
                    viewBox="0 0 448 512"
                    width="1em"
                  >
                    <path
                      d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
                    />
                  </svg>
                </span>
              </a>
            </div>
            <div
              class="pf-l-flex pf-m-space-items-lg pf-m-column pf-m-align-items-flex-start"
            >
              <div
                class="pf-l-flex pf-m-space-items-sm pf-m-grow pf-m-column pf-m-align-items-flex-start"
              >
                <span
                  class="pf-c-label pf-m-blue"
                >
                  <span
                    class="pf-c-label__content"
                  >
                    <span
                      class="pf-c-label__icon"
                    >
                      <svg
                        aria-hidden="true"
                        fill="currentColor"
                        height="1em"
                        role="img"
                        style="vertical-align: -0.125em;"
                        viewBox="0 0 512 512"
                        width="1em"
                      >
                        <path
                          d="M416 320h-96c-17.6 0-32-14.4-32-32s14.4-32 32-32h96s96-107 96-160-43-96-96-96-96 43-96 96c0 25.5 22.2 63.4 45.3 96H320c-52.9 0-96 43.1-96 96s43.1 96 96 96h96c17.6 0 32 14.4 32 32s-14.4 32-32 32H185.5c-16 24.8-33.8 47.7-47.3 64H416c52.9 0 96-43.1 96-96s-43.1-96-96-96zm0-256c17.7 0 32 14.3 32 32s-14.3 32-32 32-32-14.3-32-32 14.3-32 32-32zM96 256c-53 0-96 43-96 96s96 160 96 160 96-107 96-160-43-96-96-96zm0 128c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32z"
                        />
                      </svg>
                    </span>
                    Explore subscription benefits
                  </span>
                </span>
                <p>
                  Explore the benefits of your Red Hat subscription.
                </p>
                <ul
                  class="pf-c-list pf-m-plain"
                >
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-4"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://www.redhat.com/en/about/value-of-subscription"
                      target="_blank"
                    >
                      Benefits of a Red Hat subscription
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-5"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://www.redhat.com/en/about/subscription-model-faq"
                      target="_blank"
                    >
                      Red Hat subscription model FAQs
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                </ul>
              </div>
              <a
                aria-disabled="false"
                class="pf-c-button pf-m-link pf-m-inline"
                data-ouia-component-id="OUIA-Generated-Button-link-6"
                data-ouia-component-type="PF4/Button"
                data-ouia-safe="true"
                href="https://www.redhat.com/en/contact"
              >
                Talk to a Red Hatter
                <span
                  class="pf-c-button__icon pf-m-end"
                >
                  <svg
                    aria-hidden="true"
                    fill="currentColor"
                    height="1em"
                    role="img"
                    style="vertical-align: -0.125em;"
                    viewBox="0 0 448 512"
                    width="1em"
                  >
                    <path
                      d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
                    />
                  </svg>
                </span>
              </a>
            </div>
            <div
              class="pf-l-flex pf-m-space-items-lg pf-m-column pf-m-align-items-flex-start"
            >
              <div
                class="pf-l-flex pf-m-space-items-sm pf-m-grow pf-m-column pf-m-align-items-flex-start"
              >
                <span
                  class="pf-c-label pf-m-blue"
                >
                  <span
                    class="pf-c-label__content"
                  >
                    <span
                      class="pf-c-label__icon"
                    >
                      <svg
                        aria-hidden="true"
                        fill="currentColor"
                        height="1em"
                        role="img"
                        style="vertical-align: -0.125em;"
                        viewBox="0 0 512 512"
                        width="1em"
                      >
                        <path
                          d="M505.12019,19.09375c-1.18945-5.53125-6.65819-11-12.207-12.1875C460.716,0,435.507,0,410.40747,0,307.17523,0,245.26909,55.20312,199.05238,128H94.83772c-16.34763.01562-35.55658,11.875-42.88664,26.48438L2.51562,253.29688A28.4,28.4,0,0,0,0,264a24.00867,24.00867,0,0,0,24.00582,24H127.81618l-22.47457,22.46875c-11.36521,11.36133-12.99607,32.25781,0,45.25L156.24582,406.625c11.15623,11.1875,32.15619,13.15625,45.27726,0l22.47457-22.46875V488a24.00867,24.00867,0,0,0,24.00581,24,28.55934,28.55934,0,0,0,10.707-2.51562l98.72834-49.39063c14.62888-7.29687,26.50776-26.5,26.50776-42.85937V312.79688c72.59753-46.3125,128.03493-108.40626,128.03493-211.09376C512.07526,76.5,512.07526,51.29688,505.12019,19.09375ZM384.04033,168A40,40,0,1,1,424.05,128,40.02322,40.02322,0,0,1,384.04033,168Z"
                        />
                      </svg>
                    </span>
                    Manage users
                  </span>
                </span>
                <p>
                  Grant and manage access permissions for individuals.
                </p>
                <ul
                  class="pf-c-list pf-m-plain"
                >
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-7"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://access.redhat.com/start/how-to-create-and-manage-users"
                      target="_blank"
                    >
                      How To Create and Manage Users
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-8"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://access.redhat.com/start/how-to-create-and-manage-users#singleuser"
                      target="_blank"
                    >
                      Add a Single User
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-9"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://access.redhat.com/start/how-to-create-and-manage-users#multipleusers"
                      target="_blank"
                    >
                      Add Multiple Users
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                </ul>
              </div>
              <a
                aria-disabled="false"
                class="pf-c-button pf-m-link pf-m-inline"
                data-ouia-component-id="OUIA-Generated-Button-link-10"
                data-ouia-component-type="PF4/Button"
                data-ouia-safe="true"
                href="https://access.redhat.com/articles/1757953"
              >
                View all Roles and Permissions
                <span
                  class="pf-c-button__icon pf-m-end"
                >
                  <svg
                    aria-hidden="true"
                    fill="currentColor"
                    height="1em"
                    role="img"
                    style="vertical-align: -0.125em;"
                    viewBox="0 0 448 512"
                    width="1em"
                  >
                    <path
                      d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
                    />
                  </svg>
                </span>
              </a>
            </div>
            <div
              class="pf-l-flex pf-m-space-items-lg pf-m-column pf-m-align-items-flex-start"
            >
              <div
                class="pf-l-flex pf-m-space-items-sm pf-m-grow pf-m-column pf-m-align-items-flex-start"
              >
                <span
                  class="pf-c-label pf-m-blue"
                >
                  <span
                    class="pf-c-label__content"
                  >
                    <span
                      class="pf-c-label__icon"
                    >
                      <svg
                        aria-hidden="true"
                        fill="currentColor"
                        height="1em"
                        role="img"
                        style="vertical-align: -0.125em;"
                        viewBox="0 0 640 512"
                        width="1em"
                      >
                        <path
                          d="M622.34 153.2L343.4 67.5c-15.2-4.67-31.6-4.67-46.79 0L17.66 153.2c-23.54 7.23-23.54 38.36 0 45.59l48.63 14.94c-10.67 13.19-17.23 29.28-17.88 46.9C38.78 266.15 32 276.11 32 288c0 10.78 5.68 19.85 13.86 25.65L20.33 428.53C18.11 438.52 25.71 448 35.94 448h56.11c10.24 0 17.84-9.48 15.62-19.47L82.14 313.65C90.32 307.85 96 298.78 96 288c0-11.57-6.47-21.25-15.66-26.87.76-15.02 8.44-28.3 20.69-36.72L296.6 284.5c9.06 2.78 26.44 6.25 46.79 0l278.95-85.7c23.55-7.24 23.55-38.36 0-45.6zM352.79 315.09c-28.53 8.76-52.84 3.92-65.59 0l-145.02-44.55L128 384c0 35.35 85.96 64 192 64s192-28.65 192-64l-14.18-113.47-145.03 44.56z"
                        />
                      </svg>
                    </span>
                    Learning Subscriptions
                  </span>
                </span>
                <p>
                  Looking for learning content and subscriptions?
                </p>
                <ul
                  class="pf-c-list pf-m-plain"
                >
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-11"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://www.redhat.com/en/services/training-and-certification"
                      target="_blank"
                    >
                      Go to your learning content
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                  <li
                    class=""
                  >
                    <a
                      aria-disabled="false"
                      class="pf-c-button pf-m-link pf-m-inline"
                      data-ouia-component-id="OUIA-Generated-Button-link-12"
                      data-ouia-component-type="PF4/Button"
                      data-ouia-safe="true"
                      href="https://www.redhat.com/en/services/training-and-certification/contact-us"
                      target="_blank"
                    >
                      Contact Red Hat Training
                      <span
                        class="pf-c-button__icon pf-m-end"
                      >
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          height="1em"
                          role="img"
                          style="vertical-align: -0.125em;"
                          viewBox="0 0 512 512"
                          width="1em"
                        >
                          <path
                            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                          />
                        </svg>
                      </span>
                    </a>
                  </li>
                </ul>
              </div>
              <a
                aria-disabled="false"
                class="pf-c-button pf-m-link pf-m-inline"
                data-ouia-component-id="OUIA-Generated-Button-link-13"
                data-ouia-component-type="PF4/Button"
                data-ouia-safe="true"
                href="https://www.redhat.com/en/services/training-and-certification"
              >
                View all Training & Certification
                <span
                  class="pf-c-button__icon pf-m-end"
                >
                  <svg
                    aria-hidden="true"
                    fill="currentColor"
                    height="1em"
                    role="img"
                    style="vertical-align: -0.125em;"
                    viewBox="0 0 448 512"
                    width="1em"
                  >
                    <path
                      d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
                    />
                  </svg>
                </span>
              </a>
            </div>
          </div>
        </div>
      </div>
    </article>
  </div>
</body>
`;
